sdk/servicebus/azure-servicebus/samples/async_samples/send_and_receive_amqp_annotated_message_async.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show sending, receiving and parsing amqp annotated message(s) to a Service Bus Queue asynchronously.
+"""
+
+# pylint: disable=C0111
+
+import os
+import asyncio
+from azure.servicebus import AMQPAnnotatedMessage, AMQPMessageBodyType
+from azure.servicebus.aio import ServiceBusClient
+
+CONNECTION_STR = os.environ['SERVICE_BUS_CONNECTION_STR']
+QUEUE_NAME = os.environ["SERVICE_BUS_QUEUE_NAME"]
+
+
+async def send_data_message(sender):
+    data_body = [b'aa', b'bb', b'cc']
+    data_app_prop = {"body_type": "data"}
+    del_anno = {"delann_key": "delann_value"}
```

Comment:
Can we just name it "delivery_annotations" 
`delivery_annotations = {"annotation key": "value"}`

Reply:
fixed!